### PR TITLE
Fixing typos and rewording docs/examples/basic-requirements.md

### PR DESCRIPTION
**Pull request by MoShitrit**

### CODE REVIEW

File: docs/examples/basic-requirements.md [hunk 1]
```diff
@@ -42,18 +48,18 @@ yum -y install epel-release
 yum -y install jq
 ```
 
-**Debian7/Debian8/Debian9/Ubuntu1404lts/Ubuntu1604lts:**
+**Debian 7/8/9, Ubuntu 14.04/16.04:**
```
[the remark this answers is not in the document]
Comment (MoShitrit) [reply docs/examples/basic-requirements.md:51]: Yeah, good point.
I guess we can change that to `**Debian 8/9/10, Ubuntu 16.04/18.04/20/04:**`?

**Comment by MoShitrit** [comment]
> Looks better.
> Not 100% sure what value this doc actually brings though.
> Some of this can be linked to other docs like high availability, single-to-multimaster etc.

Hmm, yeah I see your point. Although examples could be sometimes useful as a starting point for someone who hasn't used `kops` at all before. Perhaps it's worth adding another section under `Getting Started` for `Examples`?

File: docs/examples/basic-requirements.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (MoShitrit) [reply docs/examples/basic-requirements.md]: Bumped the versions to `**Debian 8/9/10, Ubuntu 16.04/18.04/20.04:**`

[the remark this answers is not in the document]
Comment (MoShitrit) [reply docs/examples/basic-requirements.md]: Actually, @olemarkus I'm just re-reading your comment. Did you mean _completely_ removing the versions and just keeping the distro names, so `Debian/Ubuntu` without specifying versions at all?

File: docs/examples/basic-requirements.md [hunk 2]
```diff
@@ -42,18 +48,18 @@ yum -y install epel-release
 yum -y install jq
 ```
 
-**Debian7/Debian8/Debian9/Ubuntu1404lts/Ubuntu1604lts:**
+**Debian 9/10, Ubuntu 16.04/18.04/20.04:**
```
Comment (hakman) [inline docs/examples/basic-requirements.md:51]:
```suggestion
**Debian, Ubuntu:**
```

File: docs/examples/basic-requirements.md [hunk 3]
```diff
@@ -33,7 +39,7 @@ export PATH=$PATH:~/bin
 mv kubectl kops  ~/bin
 ```
 
-Finally, some of our exercises use the "jq" utility that is available on modern linux distributions. Please ensure to install it too. Some examples of how to do it:
+Finally, some of our exercises use the `jq` utility which is available on modern linux distributions. Please ensure to install it as well. Some examples of how to do it:
 
 **Centos 7:**
```
Comment (hakman) [inline docs/examples/basic-requirements.md:44]:
```suggestion
**Amazon Linux, CentOS, RHEL:**
```

File: docs/examples/basic-requirements.md [hunk 4]
```diff
@@ -33,27 +39,26 @@ export PATH=$PATH:~/bin
 mv kubectl kops  ~/bin
 ```
 
-Finally, some of our exercises use the "jq" utility that is available on modern linux distributions. Please ensure to install it too. Some examples of how to do it:
+Finally, some of our exercises use the `jq` utility which is available on modern linux distributions. Please ensure to install it as well. Some examples of how to do it:
```
Comment (hakman) [inline docs/examples/basic-requirements.md:42]: This should be enough for someone to figure out how to install `jq`. The rest of the text below can be removed.
```suggestion
Finally, some of our exercises use the `jq` utility which is available on modern linux distributions. Please ensure to [install](https://stedolan.github.io/jq/download/) it as well.

```

File: docs/examples/basic-requirements.md [hunk 5]
```diff
@@ -1,15 +1,21 @@
...
+- AWS Region set. 
+  - Throughout most of the exercises, we'll deploy our clusters in us-east-1 region (AZs: us-east-1a, us-east-1b, us-east-1c, us-east-1d, us-east-1e and us-east-1f). 
+  - For real HA at the Kubernetes API level, you need 3 masters. 
+  - For even better HA, you'd want to make sure your region of choice has at least as many AZs as the amount of masters in your cluster (i.e. 3 masters <= 3 AZs). 
+    - You can still deploy a multi-master Kubernetes cluster on regions with only 2 AZs or even 1 AZ, but that would result in more than one (and potentially all) masters in the same AZ and thus, if that AZ goes offline, you'll lose two or all your masters. 
+    - You can always check Amazon's regions and AZs availability by following this link: [AWS Global Infrastructure](https://aws.amazon.com/about-aws/global-infrastructure/). 
+  - Remember: The masters are your Kubernetes control plane. If your masters die, you lose control over your Kubernetes cluster.
+- `kubectl` & `kops` installed. This can be done by running the following commands in your terminal (assuming you are using an amd64/x86_64 linux distro):
```
Comment (hakman) [inline docs/examples/basic-requirements.md:16]:
```suggestion
  - For real HA at the Kubernetes API level, you need [3 masters](../operations/high_availability.md). 
```

File: docs/examples/basic-requirements.md [hunk 6]
```diff
@@ -33,27 +34,27 @@ export PATH=$PATH:~/bin
 mv kubectl kops  ~/bin
 ```
 
-Finally, some of our exercises use the "jq" utility that is available on modern linux distributions. Please ensure to install it too. Some examples of how to do it:
+Finally, some of our exercises use the `jq` utility which is available on modern linux distributions. Please ensure to [install](https://stedolan.github.io/jq/download/) it as well.
```
Comment (hakman) [inline docs/examples/basic-requirements.md:37]: @MoShitrit could you also delete all the text below this line?

Comment (MoShitrit) [reply docs/examples/basic-requirements.md:37]: Done 😄 
Thanks for reviewing @hakman !

Comment (hakman) [reply docs/examples/basic-requirements.md:37]: Np. Thanks for finding these.
@olemarkus does it look better now?

**Pull request merged**